clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -52,6 +58,7 @@ let autoRefresh: boolean = true;
  *         description: Signals fides.js to use the latest custom-fides.css (if available)
  *         schema:
  *           type: boolean
+ *         example: FDS-A0B1C2
```

Comment:
this feels out of place as an example of a boolean. was this included by accident?

Reply:
Oops - that was meant to be an example of property_id. Thanks!